lms/djangoapps/grades/new/course_grade.py

```diff
@@ -240,6 +312,28 @@ def _log_event(self, log_func, log_statement):
             self.student.id
         ))
 
+    def _populate_course_structure_grades(self):
+        """
```

Comment:
I suppose we could add a `ManyToMany` field linking a `PersistentCourseGrade` with its constituent `PersistentSubsectionGrades`, and `init_from_model` all of them?

That sounds like a fairly large undertaking, but it could work. Curious to hear what others think.